modules/running-distributed-data-science-workloads.adoc

```diff
@@ -0,0 +1,95 @@
+:_module-type: PROCEDURE
+
+[id="running-distributed-data-science-workloads_{context}"]
+= Running distributed data science workloads
+
+[role='_abstract']
+To configure the distributed workloads feature for your data scientists to use in {productname-short}, you must enable several components.
+
+.Prerequisites
+* You have logged in to {openshift-platform}.
+* You have the `cluster-admin` role in {openshift-platform}.
+* You have access to a Ray cluster image. For information about how to create a Ray cluster, see the link:https://docs.ray.io/en/latest/index.html[Ray documentation].
+* You have sufficient resources. In addition to the base {productname-short} resources, you need 1.1 vCPU and 1.6 GB memory to deploy the distributed workloads infrastructure.
+* You have logged in to {productname-long}.
+ifndef::upstream[]
+* If you want to use graphics processing units (GPUs), you have enabled GPU support in {productname-short}. See link:{rhodsdocshome}{default-format-url}/managing_resources/enabling-gpu-support-in-data-science_user-mgmt[Enabling GPU support in {productname-short}].
+endif::[]
+
+.Procedure
+. Log in to the {openshift-platform} web console.
+. Enable the required components, as follows:
+.. Click *Operators* -> *Installed Operators*.
+.. Search for the *Red Hat OpenShift Data Science* Operator, and click the Operator name to open the Operator details page.
+.. Click the *Data Science Cluster* tab.
+.. Click the *rhods* cluster.
+.. Click the *YAML* tab.
+.. In the `spec.components` section, ensure that the `managementState` is set correctly for the required components depending on whether the distributed workload is run from a pipeline or notebook, as shown in the following table.
++
+|===
+|Component | Pipelines | Notebooks
+
+|`codeflare`
+|`Managed`
+|`Managed`
+
+|`dashboard`
+|`Managed`
+|`Managed`
+
+|`datasciencepipelines`
+|`Managed`
+|`Removed`
+
+|`ray`
+|`Managed`
+|`Managed`
+
+|`workbenches`
+|`Removed`
+|`Managed`
+|===
+
+.. Click *Save*.
+After a short time, the components are ready.
+. Check the status of the components, as follows:
+.. In the {openshift-platform} web console, click *Workloads* -> *Deployments*.
+.. Search for the *codeflare-operator-manager* deployment, and click the deployment name to open the deployment details page.
```

Comment:
Should this be a literal, rather than bolded?

Reply:
Given that it is a UI element that the user clicks, I think it should be in bold?